From 1acf16634912d8a35690fff9d7ad2bd0a91ce576 Mon Sep 17 00:00:00 2001
From: Emil Renner Berthing <systemd@esmil.dk>
Date: Mon, 15 Sep 2014 15:29:18 +0200
Subject: [PATCH] remove unneeded error.h includes

These are the only two places where this glibc-specific
header is included. However none of the definitions in it
seem to be used, so just remove the includes.
---
 src/notify/notify.c | 1 -
 src/path/path.c     | 1 -
 2 files changed, 2 deletions(-)

diff --git a/src/notify/notify.c b/src/notify/notify.c
index 6f1c52e3fb..33933e4bf6 100644
--- a/src/notify/notify.c
+++ b/src/notify/notify.c
@@ -21,7 +21,6 @@
 
 #include <stdio.h>
 #include <getopt.h>
-#include <error.h>
 #include <errno.h>
 #include <unistd.h>
 #include <stdlib.h>
diff --git a/src/path/path.c b/src/path/path.c
index 347921a07e..37f2571fab 100644
--- a/src/path/path.c
+++ b/src/path/path.c
@@ -21,7 +21,6 @@
 
 #include <stdio.h>
 #include <getopt.h>
-#include <error.h>
 #include <errno.h>
 #include <unistd.h>
 #include <stdlib.h>
